meson.build

```diff
@@ -578,11 +578,11 @@ foreach ident : [
         ['set_mempolicy',     '''#include <sys/syscall.h>'''],  # declared at numaif.h provided by libnuma, which we do not use
         ['get_mempolicy',     '''#include <sys/syscall.h>'''],  # declared at numaif.h provided by libnuma, which we do not use
         ['getdents64',        '''#include <dirent.h>'''],       # since glibc-2.30, but check it for musl
-        ['strerrorname_np',   '''#include <string.h>'''],       # since glibc-2.32
+        ['strerrorname_np',   '''#include <string.h>'''],       # since glibc-2.32, but check it for musl
         ['mallinfo',          '''#include <malloc.h>'''],       # deprecated since glibc-2.33, but check it for musl
```

Comment:
This can be dropped with the latest bump. We only use the best version of mallinfo() after all.